sdk/textanalytics/ai-text-analytics/src/textAnalyticsClient.ts

```diff
@@ -102,7 +102,8 @@ export interface AnalyzeSentimentOptions extends TextAnalyticsOperationOptions {
    * analysis around the aspects of a product or service (also known as
    * aspect-based sentiment analysis). If set to true, the returned
    * `SentenceSentiment` objects will have property `mined_opinions` containing
-   * the result of this analysis. Only available for API version v3.1-preview.1.
+   * the result of this analysis. Only available for API versions v3.1-preview.1
```

Comment:
nit: don't think you need this comment, since this SDK now only supports v3.1-preview.2

Reply:
This is similar to https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/textanalytics/ai-text-analytics/samples/javascript/analyzeSentimentWithOpinionMining.js#L8
it should either be updated to the version or say 3.1-preview.1 and up/above/etc

Reply:
@maririos it says "and up" in the next line.

Reply:
I don't see that next line (in the sample file I posted)

Reply:
@maririos Thanks for reporting the comment in the samples. I will fix it. My previous comment is about the comment being discussed here.

Reply:
I removed those comments from here and from the samples.